logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/AssetMapper.kt

```
@@ -47,7 +46,7 @@ class AssetMapperImpl(
             uploadAssetMetadata.isPublic,
             AssetRetentionType.valueOf(uploadAssetMetadata.retentionType.name),
             // TODO: pass the md5 to the mapper so we can return Either left in case of any error
-            calcFileMd5(dataSource) ?: ""
+            calcFileMd5(dataSource) ?: TODO("handle failure")
```

i will raise another PR for this issue to avoid silent errors